src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -4856,7 +4856,7 @@ void bar()
 {
     void bar()
     {
-        (string a, string b)[] ab = new (
+        (string a, string b)[] ab = new(
```

Comment:
Not for this PR: we'll probably want to test formatting when typing `(string a, string b)[] ab = new (string a, string b)[1];`. Is the space after `new` restored once we realized we're dealing with an array creation?